src/Context/Awareness/CommentAwareContextTrait.php

```diff
@@ -0,0 +1,38 @@
+<?php
+namespace PaulGibbs\WordpressBehatExtension\Context\Awareness;
+
+trait CommentAwareContextTrait
+{
+
+    /**
+     * Create a comment.
+     *
+     * @param array $args
+     *            Set the values of the new comment.
```

Comment:
All of the PHPDoc blocks like this is ok to go on one line (soft, optional cut-off around 120 characters line-length).